application/src/main/java/org/togetherjava/tjbot/features/help/HelpThreadCreatedListener.java

```diff
@@ -58,15 +58,15 @@ public HelpThreadCreatedListener(HelpSystemHelper helper) {
 
     @Override
     public void onMessageReceived(MessageReceivedEvent event) {
```

Comment:
please mind that this is a listener dedicated to help threads only, i would suggest to create a new listener catering to project forum.

Reply:
Do you recommend that I create a new package for that listener? Or put the listener in an existing package (like the "help" package?)

Reply:
@andrew-09 Hi sorry about late response, help contains all commands related to help section. Utils is for piece of code that can be used at multiple places, i would suggest creating a new package "project" for this